src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,91 @@
-import { useState } from 'react';
+import React, { FC, useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (newMovie: Movie) => void;
+};
+
+export const NewMovie: FC<Props> = (props) => {
```

Comment:
When you have less than 2-3 parameters in props better destructure it here.
```suggestion
export const NewMovie: FC<Props> = ({ onAdd }) => {
```